src/mole_C++/utils.cpp

```diff
@@ -48,7 +68,14 @@ sp_mat Utils::spkron(const sp_mat &A, const sp_mat &B)
     return result;
 }
 */
-
+/**
+ * @brief A wrappper for implementing a sparse Kroenecker product.
+ *
+ * @param A a sparse matrix
+ * @param B a sparse matrix
+ *
+ * @note This is available in Armadillo >8.0
```

Comment:
When linking with Armadillo >8.0, which `spkron ` should you use, Armadillo's or the one provided here?
If you recommend going with Armadillo's, should you deprecate this method?

Reply:
Handled by Armadillo >=8.5. May deprecate later, but existing code uses this for now.

Reply:
I am leaving the comment as a note and a note only. Reviewing the Armadillo implementation, I think @jcorbino implemented it very well, and I am not convinced the Armadillo version is faster.